creator-node/scripts/run-tests.sh

```diff
@@ -14,6 +14,7 @@ fi
 
 export storagePath='./test_file_storage'
 export logLevel='info'
+export printSequelizeLogs=false
```

Comment:
seems helpful to leave on for tests no?

Reply:
i feel like the output is actually easier to parse now https://app.circleci.com/pipelines/github/AudiusProject/audius-protocol/6633/workflows/5dc6d853-1b68-4caf-ad5a-c1144b743c8e/jobs/49854. but if you feel otherwise i don't mind reverting